pkgs/development/python-modules/playwright/default.nix

```diff
@@ -96,6 +97,7 @@ buildPythonPackage rec {
     tests = {
       driver = playwright-driver;
       browsers = playwright-driver.browsers;
+      inherit (nixosTests) playwright-python;
```

Comment:
I am not familiar with this pattern of inheriting tests from nixosTests but it seems quite widespread so it's probably the correct way. @SuperSandro2000 do you confirm ?

Reply:
I also just got this from other examples in nixpkgs. Also need to disable the tests on darwin and, most importantly, make the chromium test work as it still times out.